nixos/modules/services/misc/snapper.nix

```diff
@@ -9,6 +9,14 @@ in
 {
   options.services.snapper = {
 
+    snapshotOnBoot = mkOption {
```

Comment:
Why not snapshot all existing configs at boot? Doing only a root snapshot on boot seems to be very specific. The name of the config also does not reflect properly that this is only applies to root.

Reply:
changed the option to snapshotRootOnBoot, if someone wants to implement snapshotting all profiles on boot then they may do it in a separate pr